.github/renovate.json

```diff
@@ -1,24 +1,6 @@
 {
-  "enabledManagers": ["helm-requirements","gradle-wrapper","terraform","regex"],
-  "labels": ["dependencies"],
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "schedule": "after 7am and before 11am every weekday",
-  "helm-requirements":
-  {
-    "fileMatch": ["\\Chart.yaml$"],
-    "aliases": {
-      "hmctspublic": "https://hmctspublic.azurecr.io/helm/v1/repo/"
-    }
-  },
-  "regexManagers": [
```

Comment:
is this in the base? it probably should be in the global one anyway

Reply:
yeah i have added it